File: 1ano/Linguagem de Programação - C/Pascalzim/Fatorial.pas
Program Pzim ;
uses crt;
var n,o,fat:INTEGER;
Begin
writeln('Calcular o Fatorial de um número qualquer solicitado pelo usuário. Sabendo que: 5! = 5 * 4* 3 * 2 * 1 = 120');
writeln('Digite um número');
readln(o);
fat := 1;
for n:=1 to o do
fat:= fat * n;

writeln(fat);
  
End.

File: 3ano/Linguagem de Programação Multiplataforma - Java/Fatorial/src/fatorial/Fatorial.java
package fatorial;


public class Fatorial {
    public static void main(String[] args) {
        int num = Integer.parseInt(args[0]);
        int i;
        int aux = num;
        
        for(i = num-1; i >0; i--){
           aux = aux * i;
        }
        System.out.println(aux);
       
        
        
        
    }
    
}
